tests/keras/layers/recurrent_test.py

```diff
@@ -907,5 +920,48 @@ def test_rnn_cell_identity_initializer(layer_class):
                           np.concatenate([np.identity(units)] * num_kernels, axis=1))
 
 
+@keras_test
+@pytest.mark.skipif((K.backend() in ['cntk']), reason='Not supported.')
```

Comment:
You can just use `if(K.backend() == 'cntk', `

Reply:
Done.